R/load_latest_forecasts_repo.R

```diff
@@ -149,11 +166,14 @@ get_forecast_file_path <- function(models, file_path, forecast_dates, latest = F
 load_forecast_files_repo <- function(file_paths,
                                      locations = NULL,
                                      types = NULL,
-                                     targets = NULL) {
-  # validate file_paths exist
-  if (is.null(file_paths) | missing(file_paths)){
-    stop("In load_forecast_files_repo, file_paths are not provided.")
-  }
+                                     targets = NULL, 
+                                     hub = c("US", "ECDC")) {
+
+  # NB: UNSURE WHICH OF THESE VERSIONS TO KEEP
+  # # validate file_paths exist
```

Comment:
I'm thinking about keeping this part to make the error message more helpful. file.exists() could handle NULL but it won't be clear to the user what causes the error.

Reply:
:+1:

Reply:
all the checks related to covidData fail on my end